libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -850,13 +852,21 @@ - (void)setupWithCredentials:(SFOAuthCredentials*) credentials {
     self.idCoordinator.delegate = self;
 }
 
-- (void)restartAuthentication {
+- (void)restartAuthentication:(BOOL)onlyIfAuthenticating {
```

Comment:
@khawkins how do you like that?
(remember it's private method only used from two places loginViewController:didChangeLoginHost: and hostListViewController:didChangeLoginHost: